.storybook/addons.js

```diff
@@ -1,4 +1,4 @@
 import "@storybook/addon-actions/register";
 import "@storybook/addon-links/register";
 import "@storybook/addon-knobs/register";
-import "storybook-addon-styled-component-theme/dist/src/register";
+import "storybook-addon-styled-component-theme/dist/register";
```

Comment:
I would put this [link](https://github.com/echoulen/storybook-addon-styled-component-theme#legacy-configuration-v52-and-v4) in the commit message detail to explain why I need to perform this change.